js/viewer/annotation/base.js

```diff
@@ -27,27 +36,6 @@ function Base(attrs){
         }
         return false;
     }
-    /**
-     * This functions removes the style attributes, appends them to a string and returns them.
-     * @return {string}
-     */
-    this.getStyle = function() {
-        var styleString = "";
-        var styleAttributes = ['fill', "stroke", "stroke", "stroke-width"];
-
-        for (var i = 0; i <  styleAttributes.length; i++) {
-            if (this._attrs[styleAttributes[i]] && this._attrs[styleAttributes[i]].length > 0) {
-                styleString += styleAttributes[i] + ':' + this._attrs[styleAttributes[i]] + ';';
-                this._attrs[styleAttributes[i]] = null;
-            }
-        }
-
-        if (styleString[styleString.length - 1] == ';') {
-            styleString = styleString.slice(0, styleString.length - 1);
-        }
-
-        return styleString;
-    }
 
     this.exportToSVG = function(){
```

Comment:
This function is still incomplete.
- You're still iterating the objects without any particular order. So your output won't be deterministic. 
- You're not properly handling the `addedProperties`.

Reply:
This has  been added.